packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -193,6 +193,81 @@ class SkiaGoldClient {
     return true;
   }
 
+  /// Executes the `imgtest init` command in the goldctl tool for tryjobs.
+  ///
+  /// The `imgtest` command collects and uploads test results to the Skia Gold
+  /// backend, the `init` argument initializes the current tryjob.
+  Future<void> tryjobInit() async {
+    final File keys = workDirectory.childFile('keys.json');
+    final File failures = workDirectory.childFile('failures.json');
+
+    await keys.writeAsString(_getKeysJSON());
+    await failures.create();
+    final String commitHash = await _getCurrentCommit();
+    final String nthCommit = await getBranchCommitCount();
+    final String pullRequest = platform.environment['CIRRUS_PR'];
+    final String cirrusTaskID = platform.environment['CIRRUS_TASK_ID'];
+
+
+    final List<String> imgtestInitArguments = <String>[
+      'imgtest', 'init',
+      '--instance', 'flutter',
+      '--work-dir', workDirectory
+        .childDirectory('temp')
+        .path,
+      '--commit', commitHash,
+      '--keys-file', keys.path,
+      '--failure-file', failures.path,
+      '--passfail',
+      '--crs', 'github',
+      '--changelist', pullRequest,
+      '--cis', 'cirrus',
+      '--jobid', cirrusTaskID,
+      '--patchset', nthCommit,
```

Comment:
@kjlubick, is this right? Or can I cut some of these arguments for the tryjob?

Reply:
Lines 222-226 lgtm for tryjobs; The others also look fine and are needed for all such `imgtest init` invocations

Reply:
Right now, this is failing:
```
Exit code 1: Skia Gold tryjobInit failure.
stdout: 
stderr: Error: unknown flag: --crs
```

Reply:
Remind me where you get goldctl?  If it's from CIPD, I probably need to update it.

Reply:
Ah! Yes, we get goldctl from CIPD.

Reply:
Updated now in CIPD